src/rider/main/resources/messages/EfCoreUiBundle.properties

```diff
@@ -0,0 +1,179 @@
+group.EfCore.Groups.Dialogs.text=Entity Framework Core
+
+#
+# Migration actions
+action.EfCore.Features.Migrations.AddMigrationAction.text=Add Migration
+action.EfCore.Features.Migrations.AddMigrationAction.description=Add new code-first migration
+
+action.EfCore.Features.Migrations.RemoveLastMigrationAction.text=Remove Last Migration
+action.EfCore.Features.Migrations.RemoveLastMigrationAction.description=Removes last created migration
+
+action.EfCore.Features.Migrations.GenerateScriptAction.text=Generate SQL Script
+action.EfCore.Features.Migrations.GenerateScriptAction.description=Generates file with SQL script generated from a set of migrations
+
+#
+# Database actions
+action.EfCore.Features.Database.UpdateDatabaseAction.text=Update Database
+action.EfCore.Features.Database.UpdateDatabaseAction.description=Update database using a selected migration as a target
+
+action.EfCore.Features.Database.DropDatabaseAction.text=Drop Database
+action.EfCore.Features.Database.DropDatabaseAction.description=Drop database used by a DbContext
+
+#
+# DbContext actions
+action.EfCore.Features.DbContext.ScaffoldDbContextAction.text=Scaffold DbContext
+action.EfCore.Features.DbContext.ScaffoldDbContextAction.description=Generates code for a DbContext and entity types for a database
+
+#
+# Dialogs commons
+section.common=Common
+section.additional.options=Additional Options
+section.build.options=Build Options
+section.execution=Execution
+button.preview=Preview
+label.none=None
+migrations.project=Migrations project:
+startup.project=Startup project:
+startup.project.missing.comment=Your project is not listed? <a href='https://plugins.jetbrains.com/plugin/18147-entity-framework-core-ui/f-a-q#why-i-cant-see-my-project-in-a-startup-projects-field'>Help</a>
+dbcontext.class=DbContext class:
+checkbox.no.build=Skip project build process (--no-build)
+build.configuration=Build configuration:
+target.framework=Target framework:
+additional.arguments=Additional arguments:
+ef.core.tools=EF Core tools:
+dialog.title.confirmation=Confirmation
+connection=Connection:
+progress.title.loading.dbcontext.classes=Loading DbContext classes...
```

Comment:
_(Minor)_ Mind, we prefer to use `…` rather than `...` in properties files. Though it sometimes doesn't work well and requires a Unicode escape sequence :(